src/scripts/main.js

```diff
@@ -3,6 +3,22 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
-// write your code here
+function addRow(arg) {
+  const table = document.querySelector('table');
+
+  for (const person of people) {
+    const newRow = table.insertRow();
+
+    newRow.innerHTML = `
+      <td>${person.name}</td>
+      <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.ceil(person.died / 100)}</td>
+    `;
+  }
+}
+
+addRow('table');
```

Comment:
The addRow function is called with a parameter 'table', but it should be called without any parameters.